.changeset/new-wolves-live.md

```
@@ -0,0 +1,8 @@
+---
+'@dexto/agent-management': patch
+'@dexto/webui': patch
+'@dexto/core': patch
+'dexto': patch
+---
+
+Added first class support for GLM and Minimax providers. Also updated setup flow with better config syncing prompts. Improved API key handling during `dexto setup` flow when switching default models.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)